### ISSUE TITLE
Broken Links - Python Tutorial | Getting Started 

### ISSUE THREAD

**Issue by TEChopra1000** [apache/incubator-mxnet#16492]
**Broken Links Found in the [Crash Course](https://mxnet.apache.org/api/python/docs/tutorials/getting-started/crash-course/index.html#)**

* previous tutorial | [found here ](https://mxnet.incubator.apache.org/api/python/docs/tutorials/getting-started/crash-course/5-predict.html)
* previous tutorial | [found here](https://mxnet.incubator.apache.org/api/python/docs/tutorials/getting-started/crash-course/6-use_gpus.html)

**Broken links found in [Moving to MXNet from Other Frameworks](https://mxnet.apache.org/api/python/docs/tutorials/getting-started/to-mxnet/index.html)**

* Caffe to MXNet box
*  Caffe to MXNet does not show up on left-menu under Moving to MXNet from Other Frameworks

**Broken links found in [PyTorch vs Apache MXNet](https://mxnet.apache.org/api/python/docs/tutorials/getting-started/to-mxnet/pytorch.html)**

A majority of links are broken on this page and so I will not list them all here. 

**Broken links found in [Gluon: from experiment to deployment](https://mxnet.apache.org/api/python/docs/tutorials/getting-started/gluon_from_experiment_to_deployment.html)**

A majority of links are broken on this page and so I will not list them all here. 

**Broken Links found in [Logistic Regression Explained](https://mxnet.apache.org/api/python/docs/tutorials/getting-started/logistic_regression_explained.html)**

All of the links in this paragraph are broken:

To work with data, Apache MXNet provides [Dataset](https://mxnet.apache.org/api/python/gluon/data.html#mxnet.gluon.data.Dataset) and [DataLoader](https://mxnet.apache.org/api/python/gluon/data.html#mxnet.gluon.data.DataLoader) classes. The former is used to provide an indexed access to the data, the latter is used to shuffle and batchify the data. To learn more about working with data in Gluon, please refer to [Gluon Datasets and Dataloaders](https://mxnet.apache.org/tutorials/gluon/datasets.html) tutorial.

I think that the link in the word Xavier further down the page is meant to point [here](https://mxnet.apache.org/api/python/docs/api/initializer/index.html#mxnet.initializer.Xavier) or [here](https://mxnet.apache.org/api/python/docs/api/initializer/index.html).

**Incorrect links found in [Logistic Regression Explained](https://mxnet.apache.org/api/python/docs/tutorials/getting-started/logistic_regression_explained.html)**

HybridSequential
Sigmoid
LogisticRegressionOutput

**Comment by ddavydenko** [comment]
@mxnet-label-bot add [Doc]

**Comment by TEChopra1000** [comment]
@aaronmarkham these links have been fixed. This ticket can be closed.
